src/main/java/com/gravity9/mongocdc/MongoChangeStreamWorker.java

```diff
@@ -100,9 +109,11 @@ public void run() {
 				if (document != null) {
 					switch (document.getOperationType()) {
 						case UPDATE ->
-							log.info("UPDATE changedFields: " + document.getUpdateDescription().getUpdatedFields().toJson());
+							log.info("UPDATE changedFields: {}", document.getUpdateDescription() == null ? NULL_STRING : toJson(document.getUpdateDescription().getUpdatedFields()));
+						case DELETE ->
+							log.info("DELETE, document key: {}", toJson(document.getDocumentKey()));
 						default ->
-							log.info("{} document: {}", document.getOperationType().name(), document.getFullDocument().toJson());
+							log.info("{} document: {}", document.getOperationType().name(), toJson(document.getFullDocument()));
```

Comment:
Do we want to log the whole JSON?
Many potential clients would prefer to avoid seeing some data in logs. 
Or should we change it to DEBUG?

Reply:
FullDocument is now logged only in case debug is enabled.